ci/build_python.sh

```diff
@@ -6,20 +6,21 @@ set -euo pipefail
 source rapids-env-update
 
 CONDA_CONFIG_FILE="conda/recipes/versions.yaml"
+export CONDA_OVERRIDE_CUDA="${RAPIDS_CUDA_VERSION}"
 
 rapids-print-env
 
 rapids-logger "Build rapids-xgboost"
 
 rapids-mamba-retry mambabuild \
-  --no-test \
+  --use-local \
   --variant-config-files "${CONDA_CONFIG_FILE}" \
   conda/recipes/rapids-xgboost
 
 rapids-logger "Build rapids"
 
 rapids-mamba-retry mambabuild \
-  --no-test \
+  --use-local \
```

Comment:
just want to mention that for other repos, we use `--channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}"`: https://github.com/rapidsai/cudf/blob/abb59c83128f956c7edcb4d7744cb0faecf0026c/ci/build_python.sh#L18-L39

`RAPIDS_CONDA_BLD_OUTPUT_DIR` is set in our CI images: https://github.com/rapidsai/ci-imgs/blob/75cad918c44c6e00480001b24cb764e1b43fa0a5/Dockerfile#L108-L113

It looks like `--use-local` works, I'm just pointing this out in case we want consistency.

Reply:
Yeah `--use-local` will check the same things. Please see [this list of paths]( https://github.com/conda/conda/blob/dd7373698d1684e8b7d648d730c6a02b5d908edb/conda/base/context.py#L505-L509 ) that Conda checks when `--use-local` is set